portfolio/index.html

```diff
@@ -0,0 +1,105 @@
+            </nav>
+        </header>
+        <div id="title" class="title_container">
+            <h1>ksenia iorzh</h1>
+            <p>junior web developer</p>
+        </div>
+        <div id="smiley_face" class="smiley_face">
+            <img src="./images/smiley_face.svg" alt="SVG Image" />
+        </div>
+        <div id="info">
+            <div id="about" class="about_container">
+                <h2>about</h2>
+                <p
+                    >i'm a junior web developer with a curiosity for building
+                    beautiful and thoughtful experiences. as i continue to
+                    explore the ever-evolving landscape of web development, i'm
+                    eager to contribute to projects that make a meaningful
+                    impact on the digital world.
+                </p>
+            </div>
+            <div id="skills" class="skills_container">
+                <h2>skills</h2>
+                <p>HTML/CSS/JS/Figma</p>
+            </div>
+            <div id="portfolio" class="portfolio_container">
+                <h2>portfolio</h2>
+                <a
+                    href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form"
+                    >survey form project</a
+                >
+            </div>
+
+            <div id="contact" class="contact_container">
+                <h2>contact</h2>
+                <ul>
+                    <li
+                        ><a
+                            href="https://www.linkedin.com/in/ksenia-iorzh-a57590145/"
+                            >linkedin</a
+                        ></li
+                    >
+                    <li
+                        ><a href="mailto:kseniaiorzh@gmail.com"
+                            >send me an email</a
+                        ></li
+                    >
+                </ul>
+            </div>
+        </div>
+        ><script>
+            document.addEventListener('DOMContentLoaded', function () {
+                const aboutLink = document.getElementById('about-link');
+                const skillsLink = document.getElementById('skills-link');
+                const portfolioLink = document.getElementById('portfolio-link');
+                const contactLink = document.getElementById('contact-link');
+
+                const aboutSection = document.getElementById('about');
+                const skillsSection = document.getElementById('skills');
+                const portfolioSection = document.getElementById('portfolio');
+                const contactSection = document.getElementById('contact');
+
+                aboutLink.addEventListener('click', function (event) {
+                    event.preventDefault();
+                    aboutSection.scrollIntoView({ behavior: 'smooth' });
+                });
+
+                skillsLink.addEventListener('click', function (event) {
+                    event.preventDefault();
```

Comment:
Although the JavaScript implementation provides smooth scrolling to sections, it's worth noting that you can achieve a similar effect using HTML alone. 

<!-- Although the JavaScript implementation provides smooth scrolling to sections, it's worth noting that you can achieve a similar effect using HTML alone. By utilizing the built-in anchor links and the CSS ":target" pseudo-class, you can create smooth scrolling within the same page. Below is a simple example of how this can be done: -->

```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <title>Smooth Scrolling Example</title>
        <link rel="stylesheet" href="styles.css" />
    </head>
    <body>
        <header>
            <nav>
                <ul>
                    <li><a href="#about">about</a></li>
                    <li><a href="#skills">skills</a></li>
                    <li><a href="#portfolio">portfolio</a></li>
                    <li><a href="#contact">contact</a></li>
                </ul>
            </nav>
        </header>
        <div id="about">
            <h2>About Section</h2>
            <!-- Content goes here -->
        </div>
        <div id="skills">
            <h2>Skills Section</h2>
            <!-- Content goes here -->
        </div>
        <div id="portfolio">
            <h2>Portfolio Section</h2>
            <!-- Content goes here -->
        </div>
        <div id="contact">
            <h2>Contact Section</h2>
            <!-- Content goes here -->
        </div>
    </body>
</html>
```

Reply:
It was like that before. However, this solution resulted with page being scrolled straight away on reloading the page, because of `href` being added to the url which I suppose is a wrong behaviour. I googled a bit and from what I understood it couldn't be fixed with **html** and **css** alone, hence I found and added this script.